lib/stigtest.pm

```diff
@@ -44,14 +47,20 @@ sub set_ds_file {
     # Set the ds file for separate product, e.g.,
     # for SLE15 the ds file is "ssg-sle15-ds.xml";
     # for SLE12 the ds file is "ssg-sle12-ds.xml"
+    # for Tumbleweed the ds file is "ssg-opensuse-ds.xml"
```

Comment:
It is good to keep the same style by the end of comments( 'comma', 'semicolons','nothing').

Reply:
semicolon added